code-challenges.test.js

```diff
@@ -21,14 +28,40 @@ const fibLength1 = 6
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("perfectSpin", () => {
+    it("takes in a number more than 2, and returns an array with the length that corresponds to the fibonacci sequence", () => {
+     expect(perfectSpin(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+     expect(perfectSpin(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
 
 // b) Create the function that makes the test pass.
 
+const perfectSpin = (num) => {
+let spin = [0, 1]
+    for(let i = 2; i <= num; i++) {
+        spin.push(spin[i-2] + spin[i-1])
+        return spin
+    }
+}
+console.log(perfectSpin(fibLength2))
+console.log(perfectSpin(fibLength2))
+
+// I couldn't figure this out, and spent a whole lot of it doing exactly what Austin told us not to do, and researching the fibonacci spiral. I ended up finding that we need an array that starts with 0, 1 because the fibonnaci spiral always starts like that. I think I'm on the right track here, but I have no idea how to finish this code. 
```

Comment:
Yes the Fibonacci spiral starts that way but the test calls for an array that starts with [1,1]. The test doesn't know about Fibonacci spirals only inputs and outputs.